docs/en/setup/ai-pipeline/README.md

```diff
@@ -0,0 +1,22 @@
+# AI Pipeline
+
+**Warning, this module is still in the ALPHA stage. This is not stable.**
+
+Pattern Recognition, Machine Learning(ML) and Artificial Intelligence(AI) are common technology to identify patterns in data. 
+This module provides a way to integrate these technologies in a standardized way about shipping the data from OAP kernel
+to 3rd party.
+
+From the industry practice, Pattern Recognition, Machine Learning(ML) and Artificial Intelligence(AI) are always overestimated,
+they are good at many things but have to run in a clear context.
+
+## How to enable
+
+ai-pipeline module is required to activate manually through `application.yml` or system environment variable `SW_AI_PIPELINE`. 
```

Comment:
This doesn't match the config in `application.yaml`, it's activated there by default

Reply:
Fixed